components/profile/ProfileModal.tsx

```diff
@@ -129,7 +131,7 @@ const portfolioQuery = gql`
             contributionRate
           }
           coordinates {
-            pointTL {
+            pointTR {
```

Comment:
The problem is that when the sidebar is opened it shift right the parcel, it can be seen even when clicking on parcel, so to sort of center it an offset is added.
I think that to really center it accounting for all parcel sizes beside the offset we need to find the center point.
Does the `coordinates` array come ordered? If yes it should be easy to fix.

Reply:
Given that the offset constant was moved to the map in this PR I think we can include it here.